orderapi/src/main/resources/application-local.properties

```diff
@@ -14,10 +14,13 @@ serviceconfiguration.url=http://host.docker.internal:8187/serviceconfiguration/
 order.experience.url=http://host.docker.internal:8084/v1/order/
 
 history.service.url=http://localhost:8189
+message.service.url=http://localhost:8181/message/
```

Comment:
You can see better way to implement this, src/main/java/fi/hel/verkkokauppa/common/configuration/ServiceUrls.java